frontend/src/component/Dashboard/RecentActivity.jsx

```diff
@@ -11,20 +11,20 @@ export const RecentActivity = props =>
         {props.t('Recent activity')}
       </div>
 
-      <div
-        className={classnames('activity__header__allread', 'btn btn-outline-primary')}
+      <button
+        className={classnames('activity__header__allread', 'btn', 'primaryColorBorder primaryColorBgHover primaryColorBorderDarkenHover')}
```

Comment:
dont use classnames() function if there are only static classes in it.
this should be declares like this: `className='activity__header__allread btn primaryColorBorder primaryColorBgHover primaryColorBorderDarkenHover'`